### PR TITLE
Fik it asset

### PR DESCRIPTION
**Pull request by Gurrit**

### CODE REVIEW

**Review by jerge — APPROVED** [review]
Yes, very good, seems yes

**Comment by ghost** [comment]
This changes the hookit banner as well as the fikit banner

**Comment by ghost** [comment]
Nevermind 😅 Just weird commits. Everything seems fine 👌

**Review by ghost — DISMISSED** [review]

**Comment by malmers** [comment]
The indentation looks off

**Comment by malmers** [comment]
First off: I love seeing non-digIT folks contribute with code, that's awesome!

It's not my call to say whether this feature should be accepted or if it will just add bloat to hubbIT. I do however have some constructive feedback on how to make the code work. As I see it there are two issues with it right now:

* The image is referenced as `FikIT.png` in the scss but is saved as `fikIT.png`
* The styling for `tr#FikIT` is accidentally nested within `tr#HookIT`

The code should probably be written as:
```
tr#HookIT {
  background: image-url("wanted.png");
  th {
    color: #282828;
    font-family: "Rye";
    font-size: 1em;
  }
}

tr#FikIT {
  background: image-url("fikIT.png");
  th {
    color: white;
    font-size: 1em;
    text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
  }
}
```

**Comment by Portals** [comment]
This should not be merged before Malms concerns are fixed!

**Comment by Kaffe-work** [comment]
Updated code in accordance to sedimentary suggestions.

File: app/assets/stylesheets/sessions.css.scss [hunk 1]
```diff
@@ -79,5 +79,13 @@
           font-family: "Rye";
           font-size: 1em;
       }
-  }
-}
+    }
```
Comment (malmers) [inline app/assets/stylesheets/sessions.css.scss:81]: Indentation issue

File: app/assets/stylesheets/sessions.css.scss [hunk 2]
```diff
@@ -79,5 +79,13 @@
...
+  tr#FikIT { 
+      background: image-url("fikIT.png");
+      th {
+          color: white;
+          font-size: 1em;
+	  text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
```
Comment (malmers) [inline app/assets/stylesheets/sessions.css.scss]: Indentation issue

File: app/assets/stylesheets/sessions.css.scss [hunk 3]
```diff
@@ -79,5 +79,13 @@
...
+      }
+    }
```
Comment (malmers) [inline app/assets/stylesheets/sessions.css.scss]: Indentation issue

File: app/assets/stylesheets/sessions.css.scss [hunk 4]
```diff
@@ -79,5 +79,13 @@
...
+}
```
Comment (malmers) [inline app/assets/stylesheets/sessions.css.scss]: No newline at end of file anymore :(

**Review by malmers — COMMENTED** [review]
So close!

File: app/assets/stylesheets/sessions.css.scss [hunk 5]
```diff
@@ -73,11 +73,19 @@
...
+      font-family: "Rye";
+      font-size: 1em;
+    }
+  }
+  tr#FikIT { 
+    background: image-url("fikIT.png");
+    th {
+      color: white;
+      font-size: 1em;
+	  text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
```
Comment (malmers) [inline app/assets/stylesheets/sessions.css.scss]: Looks like there's a hidden tab here ;)

**Comment by Kaffe-work** [comment]
When removing tabs, i checked the whole file again for indentation issues, and autoremoved tabs, turns out there are a few in the old code. I could push these changes as well.

**Review by ghost — APPROVED** [review]

**Pull request merged**